### ISSUE TITLE
Add Change Player Model Button to Spawn Menu

### ISSUE THREAD

**Issue by A1steaksa** [Facepunch/garrysmod-requests#2553]
The button to open Player Model selection is currently "hidden" in the "C menu".  
As far as I can tell, there is no specific reason that it was put in the C menu other than that it needed to go **somewhere**.

I just needed to change my player model for the first time in quite a while and I was unable to find it.  After spending more time than I should have trying to find it myself, I wound up searching online for it.  That's not a good sign for its discoverability.

I thought initially it might be in the "Game" tab of the Options menu, given that's where other player customization options exist.
Then I assumed it would be in one of the spawn menu's many, many sections and panels.  I searched through the Utilities tab, or the Server tab, or maybe even the tools tab, but no.

Instead you have to activate the "Hold the camera still while using the toolgun" mode which has literally nothing to do with player models.

It could even stay in the same location, as that's naturally in the spawn menu's left margin.

By taking screenshots and using advanced copy-and-paste technology on them, I managed to build a mockup image of what this might look like:
![Image](https://github.com/user-attachments/assets/3154b20c-7bed-4e10-8cc1-7296b6492608)

**Comment by TotallyARussian** [comment]
back in the day, before the context menu was a thing, gmod initially had playermodel settings in the spawn menu, i doubt they'd bring it back, they probably wanted to avoid clogging up the spawnmenu settings, and besides, if they did bring it to the spawn menu new players might not even come across the context menu, because they wouldn't be searching for a way to change the player model via the context menu if they did it via the spawn menu, so they'd potentially miss out on plethora of useful features for an unknown amount of time.

tl;dr: i think the playermodel menu being in the context menu helps new players to find out about the context menu.

**Comment by A1steaksa** [comment]
If the C menu is under-utilized or difficult for new users to discover on their own, that should be solved with something like a hint popup or a note in the spawn menu.

Attaching unrelated buttons to it doesn't encourage users to discover the C menu, it encourages users to get frustrated with Garry's Mod and stop playing.

**Comment by robotboy655** [comment]
I have been recently thinking about this as well, it is too hidden.

Added a button for it here:
![Image](https://github.com/user-attachments/assets/cd7b856a-8151-402c-88c9-cf6387db8a72)

Honestly took too much time to get it working due to how those context menu widgets are set up, but it is done now.
